OpenSim/Simulation/Model/ContactMesh.cpp

```diff
@@ -37,7 +37,7 @@ ContactMesh::ContactMesh() :
     setupProperties();
 }
 
-ContactMesh::ContactMesh(const std::string& filename, const SimTK::Vec3& location, const SimTK::Vec3& orientation, Body& body) :
+ContactMesh::ContactMesh(const std::string& filename, const SimTK::Vec3& location, const SimTK::Vec3& orientation, PhysicalFrame& body) :
```

Comment:
Either way; not a big deal. I just thought you meant to change `body` to `frame` here because you changed it in the header file. :snowman: